_data/versions.yml

```diff
@@ -1,3 +1,103 @@
+v3.1:
+- title: v3.1.0-rc1
+  note: |
+  18 March 2018
+
```

Comment:
The indentation is off such that Jekyll crashes.....need to add two spaces in front of `xx March` and the lines below it to resolve this issue.